### DISCUSSION TITLE
ISR cache issue, GetStaticProps some time not revalidate

### DISCUSSION THREAD

**Discussion by nchaintrade** [vercel/next.js#32782] [Help]
### What version of Next.js are you using?

12.0.3

### What version of Node.js are you using?

14.15

### What browser are you using?

chrome, firefox, safari

### What operating system are you using?

macOs, Ubuntu, Windows

### How are you deploying your application?

jenkins, docker container

### Describe the Bug

I have 10 hard server side requests. I use getStaticProps with revalidation in 3 minutes, after a while, the data that I request from the back is updated, but nothing changes on the client, taking into account that I get the correct page.json with props.
I also use localization next-i18next. 
If i used serverSideProps - no problem. This issue reproduce on 12.0.3 and previous too

![Снимок экрана 2021-11-10 в 17 20 01](https://user-images.githubusercontent.com/38041874/141142252-b11943a7-09e1-47d2-bdca-f7446f93dada.png)

### Expected Behavior

after revalidate data on client must update

### To Reproduce

create a page with gsp, and after 1-2 hours update data, and try revalidate

**Comment by nchaintrade** [comment]
![Снимок экрана 2021-11-10 в 18 19 53](https://user-images.githubusercontent.com/38041874/141151338-e7032601-0031-429c-ba5a-0362f96c0929.png)

**Comment by jorgev259** [comment]
Having the same problem on a manually built Ubuntu machine using `next start` with nginx. Using revalidate: 60 doesnt update the page

**Comment by VitaliyInshakov** [comment]
Yep, I have this problem too((

**Comment by mattjis** [comment]
Seems to be fixed for me in upcoming version 12.0.4-canary.5

https://github.com/vercel/next.js/issues/30498

**Comment by vechnii2007** [comment]
> Кажется, это будет исправлено для меня в следующей версии 12.0.4-canary.5
> 
> [# 30498](https://github.com/vercel/next.js/issues/30498)

for me everything works, but after a while the re-check stops for the page on the same locale (en), if I change the locale and wait for the timeout, re-check, my page will refresh, but if I go back to the language when the re-check stopped, mine the page is not refreshed

**Comment by hacknlove** [comment]
Still not working.

It works locally with `npm run build; npm run start" but it does not work at vercel
